### ISSUE TITLE
twisted.internet.test.test_pollingfile fails on Windows Python 3

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#8733]
|[<img alt="rodrigc's avatar" src="https://avatars.githubusercontent.com/u/1895943?s=50" width="50" height="50">](https://github.com/rodrigc)| @rodrigc reported|
|-|-|
|Trac ID|trac#8733|
|Type|enhancement|
|Created|2016-08-08 01:09:17Z|
|Branch|https://github.com/twisted/twisted/tree/8733-rodrigc-pollingfile-py3|

On Windows, I do the following in a Python 3 environment:

```
tox -e py35-tests-windows twisted.internet.test.test_pollingfile
```

It fails.

<details><summary>Searchable metadata</summary>

```
trac-id__8733 8733
type__enhancement enhancement
reporter__rodrigc rodrigc
priority__normal normal
milestone__Python_3_x Python-3.x
branch__8733_rodrigc_pollingfile_py3 8733-rodrigc-pollingfile-py3
branch_author__ 
status__closed closed
resolution__fixed fixed
component__core core
keywords__None None
time__1470618557688401 1470618557688401
changetime__1474198049187875 1474198049187875
version__None None
owner__GitHub__noreply_____ GitHub <noreply@...>

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="rodrigc's avatar" src="https://avatars.githubusercontent.com/u/1895943?s=50" width="50" height="50">](https://github.com/rodrigc)<a name="note_2"></a>|@rodrigc commented|
|-|-|

https://github.com/twisted/twisted/pull/477

Tested with:

```
tox -e py35-tests-windows twisted.internet.test.test_pollingfile
```

**Comment by twisted-trac** [comment]
|<img alt="GitHub's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50"><a name="note_3"></a>|GitHub set owner to GitHub<br>GitHub set status to `closed`|
|-|-|

In changeset 5cfcea35a865bba6152b5dd51bc82a4cfb9e6898
```
#!CommitTicketReference repository="" revision="5cfcea35a865bba6152b5dd51bc82a4cfb9e6898"
Merge pull request #477 from twisted/8733-rodrigc-pollingfile-py3

Author: hawkowl, rodrigc
Reviewer: alex
Fixes: #8733

Fix twisted.internet.test.test_pollingfile on Python 3
```

**Comment by twisted-trac** [comment]
|[<img alt="hawkowl's avatar" src="https://avatars.githubusercontent.com/u/3307100?s=50" width="50" height="50">](https://github.com/hawkowl)<a name="note_4"></a>|@hawkowl commented|
|-|-|

[mass edit] Removing review from closed tickets.
